### PR TITLE
Clusterer refactor

### PR DESCRIPTION
**Pull request by Adamj1232**
## What's this PR do?

Applied clustering to my map display component and did some refactoring in MapDisplay

## Pre-Merge TODOs
 - [x] Screenshots Included
    - ANY change to the UI of the app must have screenshots/gifs
 - [ ] 2 PR reviews completed

## Learning

Learning to use mapClusterer was a pain as there is not much documentation on it for react but got it working

## Where should the reviewer start?

mapDisplay component

## Screenshots (if appropriate)
* GIF of app in use

### CODE REVIEW

File: src/components/MapDisplay.js [hunk 1]
```diff
@@ -2,35 +2,40 @@ import React from 'react'
...
+    cluster = false
   }
 
   if( searchResults.length !== 0){
     const sortedSearch = searchResults.sort( (venueA, venueB) => {
       return venueB.cityVenues.length - venueA.cityVenues.length
     })
     zoomIndex = 8
-    mapCenter = {lat:sortedSearch[0].Latitude , lng: sortedSearch[0].Longitude }
+    mapCenter = { lat:sortedSearch[0].Latitude , lng: sortedSearch[0].Longitude }
```
Comment (JohnBinning) [inline src/components/MapDisplay.js]: space after lat: => "lat: sortedSearch..."

File: src/components/MapDisplay.js [hunk 2]
```diff
@@ -55,15 +60,28 @@ const MapDisplay = withGoogleMap(({ searchResults, searchFromMap, mapPinFilter,
...
+  const clusterer = (
+    <MarkerClusterer
+      averageCenter
+      enableRetinaIcons
+      gridSize={25}
+      maxZoom={7}
+      >
+        {venuePins}
+    </MarkerClusterer>
+  );
```
Comment (JohnBinning) [inline src/components/MapDisplay.js:72]: looks like you're using a ; on 72 but not on 61.  Or I haven't used them in so long I don't actually remember where they go.  Either way.

**Comment by JohnBinning** [comment]
Looks really good dude.  Loving the clusters.

**Review by DBULL7 — COMMENTED** [review]
Looks great man, super cool that you got it working with poor documentation 🔥

**Pull request merged**